packages/objectloader2/src/operations/database.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, test } from 'vitest'
+import CacheDatabase from './database.js'
+import indexeddb from 'fake-indexeddb'
+import { Item } from '../types/types.js'
+
+globalThis.indexedDB = indexeddb
```

Comment:
Maybe indexedDb should be injected through constructor options too? It could default to globalThis.indexedDB, but could also be overridden through ctor options